.circleci/config.yml

```diff
@@ -90,8 +90,11 @@ jobs:
           key: feign-dependencies-{{ checksum "pom.xml" }}
       - run:
           name: 'Test'
+          # Do not generale module-info when running tests
+          # because one of the current JDK targets is 8
+          # Moditect requires 11 as a minimum
           command: |
-            ./mvnw -ntp -B test
+            ./mvnw -ntp -B test -Dmoditect.skip=true
```

Comment:
Think we can revert this now, right?!

Reply:
Please revert this, build now runs using java 17 and just compiler brings java 8 to the mix.

Reply:
Will do.